00_Base/src/datasources/CdrsDatasource.ts

```diff
@@ -0,0 +1,29 @@
+import { Inject, Service, Token } from 'typedi';
+import { PaginatedResult } from '../model/PaginatedResult';
+import { ITransactionDatasource } from './ITransactionDatasource';
+import { TRANSACTION_DATASOURCE_SERVICE_TOKEN } from './TransactionDatasource';
+import { ICdrsDatasource } from './ICdrsDatasource';
+import { CdrMapper } from '../mapper/cdr.mapper';
+import { Cdr } from '../model/Cdr';
+
+export const CDR_DATASOURCE_SERVICE_TOKEN = new Token('CDR_DATASOURCE_SERVICE_TOKEN');
+
+@Service(CDR_DATASOURCE_SERVICE_TOKEN)
+export class CdrsDatasource implements ICdrsDatasource {
```

Comment:
Can leave as is for now, but in future we might want to consider renaming to something like `CdrsDatasourceOcpp2_0_1`.

Thinking ahead, for a network that is both 1.6 and 2.0.1, I'm just wondering how the get pagination logic would work since it would need to fetch and merge both 1.6 and 2.0.1 data. Seems like it will be tricky.

Not something we need to worry about rn though.